roles/advanced-core/advanced_dhcp_server/templates/dhcpd.subnet.conf.j2

```diff
@@ -32,14 +32,15 @@
       {% set ipxe_rom_filename = (ipxe_architecture + '/standard_' + ipxe_driver + 'ipxe.efi') %}
     {% else %}  {# Assume everything else is pcbios #}
```

Comment:
Trailing whitespaces are between the {% else %} block and the comment block. Remove the 2 spaces before each comment and the destination file should look better.